### PR TITLE
nikolai.berestevich/fix/p2p-page-hero-2-time

### PR DESCRIPTION
**Pull request by nikolaiberestevich-deriv**
Changes:

-   bug of the second hero-component  is fixed

## Type of change

-   [ x ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by kevinw-binary — DISMISSED** [review]

**Comment by kevinw-binary** [comment]
LGTM

File: src/pages/p2p/index.js [hunk 1]
```diff
@@ -49,26 +51,38 @@ const DP2P_CONTENT = [
...
-            <Availability />
-            <P2PBanner title={localize('Hassle-free deposits and withdrawals')} image_name="DP2P" />
+
+            {is_mounted && (
+                <ContentWrapper>
+                    <Hero
+                        title={localize('Hassle-free deposits and withdrawals')}
+                        content={
+                            <Localize translate_text="Can’t make deposits and withdrawals in your local currency? No problem. Exchange with fellow traders in minutes." />
+                        }
```
Comment (sean-binary) [inline src/pages/p2p/index.js:73]: If you're changing the text string here, you'll need to branch off Crowdin.

**Review by nikolaiberestevich-deriv — COMMENTED** [review]
done

File: src/pages/p2p/index.js [hunk 2]
```diff
@@ -49,26 +51,38 @@ const DP2P_CONTENT = [
...
-                image_name="DP2P"
-            />
-            <DP2P reverse P2P={DP2P_CONTENT} />
-            <Numbers />
-            <ExchangeSteps />
-            <Availability />
-            <P2PBanner title={localize('Hassle-free deposits and withdrawals')} image_name="DP2P" />
+
+            {is_mounted && (
+                <ContentWrapper>
```
Comment (sean-binary) [inline src/pages/p2p/index.js:68]: Can we use `<></>` React Fragment for this?

**Review by sean-binary — APPROVED** [review]

**Pull request merged**